packages/block-library/src/paragraph/style.scss

```diff
@@ -35,3 +35,8 @@ p.has-background {
 p.has-text-color a {
 	color: inherit;
 }
+
+// Prevent an empty P tag from collapsing, so it matches the backend.
+p:empty::before {
```

Comment:
Out of curiosity, why `::before` and not `:before`?

Reply:
Linter says `::` is the correct one. I believe the single colon is CSS2 syntax, the double colon is CSS3 syntex.

Reply:
See https://developer.mozilla.org/en-US/docs/Web/CSS/::before#syntax

Reply:
wow, I didn't know that... thanks!